### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by jeanbez**
Update documentation with both code and design.

Preview available at https://pdc.readthedocs.io/en/code-docs

### CODE REVIEW

**Review by houjun — APPROVED** [review]
LGTM

**Comment by houjun** [comment]
@jeanbez, the width of the content on the pages seems to be fixed, and looks a bit narrow to me as it only covers the left half of the window if I view it full screen. Is this intended?

**Comment by jeanbez** [comment]
@houjun, that is the default width for most readthedocs templates, but I believe we could increase it as needed. I would not recommend making it fully wide as it might look strange. Maybe we can double the current width?

**Comment by houjun** [comment]
There are some code blocks that are just a bit too long for the current width, I'm debating whether to do more line breaks or increase the content width by something like 20%. What do you think?

**Comment by jeanbez** [comment]
Hmm, got it. I personally prefer to have commands broken into multiple lines; it improves readability. But I can give it a try by increasing it by 20% to see if that improves.

**Comment by jeanbez** [comment]
@houjun, can you give it another try now? If you do not see any changes, please force refresh the page to refresh the CSS.

I believe it looks better now, but there are still some commands that would benefit from being split into multiple lines.

**Comment by houjun** [comment]
Yes, once we merge the other pull request (which I added line breaks) to this branch then everything should fit within the page width.

File: README.md [hunk 1]
```diff
@@ -1,230 +1,41 @@
 [![linux](https://github.com/hpc-io/pdc/actions/workflows/linux.yml/badge.svg?branch=stable)](https://github.com/hpc-io/pdc/actions/workflows/linux.yml)
-# Proactive Data Containers (PDC)
+![GitHub release tag(latest by date)](https://img.shields.io/github/v/tag/hpc-io/pdc)
+![Spack](https://img.shields.io/spack/v/pdc)
+## Proactive Data Containers (PDC)
 Proactive Data Containers (PDC) software provides an object-centric API and a runtime system with a set of data object management services. These services allow placing data in the memory and storage hierarchy, performing data movement asynchronously, and providing scalable metadata operations to find data objects. PDC revolutionizes how data is stored and accessed by using object-centric abstractions to represent data that moves in the high-performance computing (HPC) memory and storage subsystems. PDC manages extensive metadata to describe data objects to find desired data efficiently as well as to store information in the data objects.
```
Comment (sbyna) [inline README.md]: Suggested text:
Proactive Data Containers (PDC) software provides an object-focused data management API, a runtime system with a set of scalable data object management services, and tools for managing data objects stored in the PDC system. The PDC API allows efficient and transparent data movement in complex memory and storage hierarchy. The PDC runtime system performs data movement asynchronously and provides scalable metadata operations to find and manipulate data objects. PDC revolutionizes how data is managed and accessed by using object-centric abstractions to represent data that moves in the high-performance computing (HPC) memory and storage subsystems. PDC manages extensive metadata to describe data objects to find desired data efficiently as well as to store information in the data objects.

Comment (sbyna) [reply README.md]: This same text can go to the readthedocs page.

File: README.md [hunk 2]
```diff
@@ -1,230 +1,41 @@
...
-    ```
-
-  * Run 64 client processes that concurrently create 1000 objects in total:
-    ```bash
-    srun -N 4 -n 64 -c 2 --mem=25600 --cpu_bind=cores --gres=craynetwork:1 --overlap ./bin/create_obj_scale -r 1000
-    ```
-
-
-
+You can access the full documentation of PDC installation instructions and examples at [pdc.readthedocs.io](https://pdc.readthedocs.io/en/code-docs)
```
Comment (sbyna) [inline README.md]: Suggested text:
"Full documentation of PDC with installation instructions, code examples for using PDC API, and research publications are available at [pdc.readthedocs.io](https://pdc.readthedocs.io/en/code-docs).

**Review by sbyna — CHANGES_REQUESTED** [review]
Added a couple of blocks of text that needs to be changed.

File: README.md (hunk 2 above)
Comment (sbyna) [reply README.md]: Is this going to be in the /code-docs folder or the main?

Comment (jeanbez) [reply README.md]: ops, main, let me fix that

File: README.md (hunk 1 above)
Comment (jeanbez) [reply README.md]: done!

File: README.md [hunk 3]
```diff
@@ -1,230 +1,42 @@
...
+Full documentation of PDC with installation instructions, code examples for using PDC API, and research publications are available at [pdc.readthedocs.io](https://pdc.readthedocs.io)
+
+More information and publications on PDC is available at https://sdm.lbl.gov/pdc
+
+If you use PDC in your research, please use the following citations:
+
+```
+@misc{byna:2017:pdc,
+  title = {Proactive Data Containers (PDC) v0.1},
+  author = {Byna, Suren and Dong, Bin and Tang, Houjun and Koziol, Quincey and Mu, Jingqing and Soumagne, Jerome and Vishwanath, Venkat and Warren, Richard and Tessier, François and USDOE},
```
Comment (sbyna) [inline README.md]: Sorry for noticing this now. Can remove USDOE at the end.

Comment (jeanbez) [reply README.md]: no problem, done!

**Review by sbyna — APPROVED** [review]

**Pull request merged**